### DISCUSSION TITLE
关于pcl2的首页的皮肤头像渲染的一些想法（雾

### DISCUSSION THREAD

**Discussion by E0x72-21** [Meloong-Git/PCL#3313] [提问] [answered]
话说pcl2的皮肤头像渲染是怎么搞的？龙猫可以单独搞一个皮肤头像渲染的小工具吗？(可以导出高分辨率图片的那种

**Comment by WForst-Breeze** [comment, marked as answer]
没找错的话应该源码是这段
https://github.com/Hex-Dragon/PCL2/blob/8c8b1066bff6476a14437a14bc16a281e25145bd/Plain%20Craft%20Launcher%202/Pages/PageLaunch/MySkin.xaml.vb#L74-L128

单独搞个小工具感觉不太可能（
